### DISCUSSION TITLE
go-acc support with go-junit-report

### DISCUSSION THREAD

**Discussion by nitsharma44** [jstemmer/go-junit-report#133] [Q&A]
Hi, 
I tried to use go-junit-report with go-acc (not go test) and it seems like i was able to generate a correct report.xml in local
but it seems like when I used the same in jenkins environment the report.xml file is missing the class names  . 

Can someone suggest the cause of this problem or some other workaround .

**Comment by jstemmer** [comment]
I've made some changes to the naming in the v2 release of go-junit-report, are the versions on your local machine and in the jenkins environment the same?

It that's not the cause of the problem, let me know what version you're using, the go-acc output that gets passed to go-junit-report and the resulting report.xml.
